trimmer_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimDecorator < Decorator
```

Comment:
- [ ] Kindly import the `base_decorator` here and also rename this parent class to match the class you defined, use the suggestion above for hint.